db/migrate/20170504213048_remove_postal_code_from_targets.rb

```diff
@@ -0,0 +1,23 @@
+class RemovePostalCodeFromTargets < ActiveRecord::Migration
+  class Call < ActiveRecord::Base; end
+  class Plugins::CallTool < ActiveRecord::Base; end
+
+  def up
+    batch_size = 300
+    Call.find_in_batches(batch_size: batch_size) do |group|
+      group.each do |call|
+        target = call.read_attribute(:target)
+        target.delete('postal_code')
+        call.update!(target: target)
+      end
+    end
+
+    Plugins::CallTool.find_in_batches(batch_size: batch_size) do |group|
+      group.each do |call_tool|
+        targets = call_tool.read_attribute(:targets)
+        targets.each { |t| t.delete('postal_code') }
+        call_tool.update!(targets: targets)
+      end
+    end
+  end
+end
```

Comment:
I see your point, but I don't think that Migrations should only be used for schema changes only. I think migrations are a great place to run data format migrations like these. What is usually recommended is not to depend on models or clases from the source code (ie AR models), because they can be removed later making the migration to stop working. That's why I redefine them in the migration file to make sure they exist. 

Anyway, this is not a big deal for me anyway, I'd just like to get this deployed. If you feel I should remove this I'll do it and I'll run the script from the console after the deploy. We should also let everyone in the team that they'll need to run this script locally after updating.